File: app/src/main/java/blue/koenig/kingsfinances/view/FinanceNullView.java
package blue.koenig.kingsfinances.view;

import android.support.annotation.StringRes;

import com.koenig.commonModel.User;
import com.koenig.commonModel.finance.BankAccount;
import com.koenig.commonModel.finance.Expenses;
import com.koenig.commonModel.finance.StandingOrder;

import java.util.List;

import blue.koenig.kingsfinances.model.calculation.Debts;

/**
 * Created by Thomas on 19.10.2017.
 */

public class FinanceNullView implements FinanceView {
    @Override
    public void askForNameOrImport() {

    }

    @Override
    public void askJoinOrCreateFamily() {

    }

    @Override
    public void showText(String text) {

    }

    @Override
    public void showText(@StringRes int stringResource) {

    }

    @Override
    public void showConnectionStatus(boolean connected) {

    }

    @Override
    public void showExpenses(List<Expenses> expenses) {

    }

    @Override
    public void setFamilyMembers(List<User> members) {

    }

    @Override
    public void showStandingOrders(List<StandingOrder> standingOrders) {

    }

    @Override
    public void updateBankAccounts(List<BankAccount> bankAccounts) {

    }

    @Override
    public void updateDebts(List<Debts> debts) {

    }
}

File: app/src/main/java/blue/koenig/kingsfinances/view/FinanceNullView.kt
package blue.koenig.kingsfinances.view

import android.support.annotation.StringRes

import com.koenig.commonModel.User
import com.koenig.commonModel.finance.BankAccount
import com.koenig.commonModel.finance.StandingOrder
import com.koenig.commonModel.finance.statistics.MonthStatistic

import com.koenig.commonModel.finance.statistics.StatisticEntryDeprecated
import org.joda.time.YearMonth

/**
 * Created by Thomas on 19.10.2017.
 */

class FinanceNullView : FinanceView {
    override fun updateAssets(assets: Map<YearMonth, MonthStatistic>) {

    }

    override fun askForNameOrImport() {

    }

    override fun askJoinOrCreateFamily() {

    }

    override fun showText(text: String) {

    }

    override fun showText(@StringRes stringResource: Int) {

    }

    override fun showConnectionStatus(connected: Boolean) {

    }


    override fun setFamilyMembers(members: List<User>) {

    }

    override fun showStandingOrders(standingOrders: List<StandingOrder>) {

    }

    override fun updateBankAccounts(bankAccounts: List<BankAccount>) {

    }

    fun updateAssets(assets: List<StatisticEntryDeprecated>) {

    }

}
